nsone/__init__.py

```diff
@@ -85,6 +86,15 @@ def datafeed(self):
         import nsone.rest.data
         return nsone.rest.data.Feed(self.config)
 
+    def monitorsjob(self):
```

Comment:
I think it makes sense to call this method `monitors`, following the returned classes name. Thoughts?